pages/chaoss/visualizations/contrib_activity_cycle.py

```diff
@@ -7,15 +7,15 @@
 import logging
 from dateutil.relativedelta import *  # type: ignore
 import plotly.express as px
-from pages.utils.graph_utils import get_graph_time_values, color_seq
+from pages.utils.graph_utils import color_seq
 from queries.commits_query import commits_query as cmq
 import io
 from cache_manager.cache_manager import CacheManager as cm
 from pages.utils.job_utils import nodata_graph
 import time
 
-PAGE = "chaoss"  # EDIT FOR PAGE USED
-VIZ_ID = "contrib-activity-cycle"  # UNIQUE IDENTIFIER FOR CALLBAKCS, MUST BE UNIQUE
```

Comment:
"CALLBAKCS" -> "CALLBACKS"
probably need to refactor across all files using vscode.

Reply:
This line was deleted not added